lib/init.js

```diff
@@ -127,14 +127,9 @@ function initTestium(config) {
       return config.get('proxy.targetUrl') + '/testium-priming-load';
     }
 
-    function getChromeDevtoolsPort() {
-      return config.get('chrome.remoteDebuggingPort', null);
-    }
-
     testium = {
       close: close,
       config: config,
-      getChromeDevtoolsPort: getChromeDevtoolsPort,
```

Comment:
This would break https://github.com/testiumjs/testium-driver-wd/blob/master/lib/testium-driver-wd.js#L169 . Either we need to revert there too, or leave this file as is